src/adapters/poi/idunn_poi.js

```diff
@@ -103,6 +103,10 @@ export default class IdunnPoi extends Poi {
     } catch (err) {
       if (err === 404) {
         return;
+      } else if (err === 0) { // When the OPTIONS request is rejected, the error is 0 and not 405
```

Comment:
Just as small syntax preference, but in cases of early returns like here I find it simpler to not use `else` constructs at all.

Reply:
Takes more place if not. :p

Reply:
So?
Ok, I'll push another ESLint rule later for that :P

Reply:
You make me sad sometimes haha.